Dockerfile.dev

```diff
@@ -1,14 +1,9 @@
-FROM ubuntu:latest
-
-RUN apt update && apt upgrade
-RUN apt install python3.10 pip -y
+FROM python:3.10.9
+RUN apt-get update && pip install --upgrade pip
```

Comment:
don't we need to call `apt-get upgrade` after update?
No idea

Reply:
No not in a container. I don't know why though :upside_down_face: 
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Reply:
Oh wow tks for the link it's quite useful!
After some reflexion I can guess why: update downloads new packages and upgrade installs them if I remember correctly. But because the image is shipped upgraded, upgrade is useless, however `update` does something else: it updates the mirrors/repo location/DB (not sure about the right terms here) but we can see it in `apt update` output (same idea when pacman synx the packages db

Reply:
However you don't **NEED** to use `apt-get update` here because you don't install anything (pip has different sources) am I right ?

Reply:
:heavy_check_mark: